backend/src/main/java/com/kodstar/backend/model/enums/Label.java

```diff
@@ -1,4 +1,6 @@
 package com.kodstar.backend.model.enums;
 
 public enum Label {
+
+    TODO, STORY
```

Comment:
1. modify enums with values like TODO("todo")
2. insert labels' names as described --> labels (fixed set of values: bug, enhancement, backend, frontend, test)